packages/default-theme/components/SwProductCard.vue

```diff
@@ -72,16 +72,20 @@ export default {
       return getProductUrl(this.product)
     },
     getRegularPrice() {
+      // remove that logic once the SW6 API returns right data
```

Comment:
please add `TODO:` in these comments, to be easily tracked; great that issues are linked <3

```suggestion
      // TODO: remove that logic once the SW6 API returns right data
```

Reply:
:+1: